crypto/src/main/java/org/web3j/crypto/Bip32ECKeyPair.java

```diff
@@ -117,7 +117,7 @@ private Bip32ECKeyPair deriveChildKey(int childNumber) {
             Arrays.fill(i, (byte) 0);
             BigInteger ilInt = new BigInteger(1, il);
             Arrays.fill(il, (byte) 0);
-            BigInteger privateKey = getPrivateKey().add(ilInt).mod(Sign.CURVE.getN());
+            BigInteger privateKey = getPrivateKey().add(ilInt).mod(ECDSASignature.K1_CURVE.getN());
```

Comment:
It should be implemented in some generic way, without using some specific realization.

Reply:
I think best if i put Sign.CURVE.getN() back here. I believe bip32ECkeys are always in k1, we might not need to generify here at all. Im going to put back how it was before